sdk/monitor/monitor-query/package.json

```diff
@@ -0,0 +1,150 @@
+    "build:types": "downlevel-dts types/latest types/3.1",
+    "build": "tsc -p . && npm run lint && npm run build:nodebrowser && api-extractor run --local && npm run build:types",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* temp types *.tgz *.log coverage coverage-browser",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src",
+    "execute:samples": "echo Obsolete",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "npm run generate:client:logquery && npm run generate:client:metrics && npm run generate:client:metrics-namespaces && npm run generate:client:metrics-definitions",
+    "generate:client:logquery": "cd swagger && autorest --typescript --tag=logquery README.md",
+    "generate:client:metrics": "cd swagger && autorest --typescript --tag=metrics README.md",
+    "generate:client:metrics-namespaces": "cd swagger && autorest --typescript --tag=metrics-namespaces README.md",
+    "generate:client:metrics-definitions": "cd swagger && autorest --typescript --tag=metrics-definitions README.md",
+    "integration-test:browser": "echo 'karma start --single-run' temporarily disabled",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --slow 5000 --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}/*.spec.js\"",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --format unix --ext .ts",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run build:test && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run integration-test:node",
+    "test": "npm run build:test && npm run integration-test",
+    "unit-test:browser": "echo 'karma start --single-run' temporarily disabled",
+    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/internal/unit/**/*.spec.ts\"",
+    "unit-test": "npm run build:test && npm run unit-test:node && npm run unit-test:browser"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/latest/monitor-query.d.ts",
+    "types/3.1/monitor-query.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": "github:Azure/azure-sdk-for-js",
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
Match the README?
```suggestion
    "node": ">=10.0.0"
```

Reply:
(undone, for now). We haven't changed up the eslint rule yet and I don't want to add in an exclusion until we're ready to do it more broadly.